src/ext2020/README.md

```diff
@@ -1,37 +0,0 @@
-## Live broadcast settings
```

Comment:
Please modify this readme and move to `attendee` app

Reply:
OK